services/user-svc/README.md

```diff
@@ -1,4 +1,3 @@
 # user-svc
 
 > User management service
-
```

Comment:
I am a big fan of trailing new lines, looks like your IDE isn't ^^

Reply:
Mh, weird. My IDE + editorconfig doesn't complain, and they show me a newline on my end ... when you pull?